sat/cli/bootsys/platform.py

```diff
@@ -464,6 +469,16 @@ def do_kubelet_start(ncn_groups):
                                target_state='active', target_enabled='enabled')
 
 
+def do_recreate_cronjobs(_):
+    """Recreate cronjobs that are not being scheduled on time."""
+    LOGGER.info('Recreating cronjobs which have failed to have been scheduled on time.')
```

Comment:
This log message is redundant with the info-level log message emitted on line 616 in `do_platform_action`.